cmd/minishift/cmd/image/config_view.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"github.com/spf13/cobra"
+
+	"fmt"
+	"github.com/minishift/minishift/cmd/minishift/cmd/config"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"sort"
+	"strings"
+)
+
+var (
+	listConfiguredImagesCmd = &cobra.Command{
+		Use:   "view",
+		Short: "Displays the configured list of images for import and export.",
+		Long:  "Displays the configured list of images for import and export.",
+		Run:   viewConfiguredImages,
+	}
+)
+
+func viewConfiguredImages(cmd *cobra.Command, args []string) {
```

Comment:
Duplicated functionality to handle reading of config.

Reply:
not quite sure I see the duplication here.

Reply:
```
+	configMap, err := config.ReadConfig()
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Cannot read the Minishift configuration: %s", err.Error()))
+	}
```
Reading and erroring out is duplicated.

Reply:
I changed the implementation a bit. So it is a bit cleaner now.

Reply:
I hope you understand, because you already agree that you use the `cmd/config` logic, so why doesn'y this live on that level in the first place. These can be used for other commands. Such as #1497